blueprints/ember-cli-blanket/index.js

```diff
@@ -35,31 +33,6 @@ module.exports = {
               '    <style>#blanket-main { position: relative; z-index: 99999; }</style>',
               { after: '<link rel="stylesheet" href="assets/test-support.css">' + EOL }
           );
-        }.bind(this))
-
-        .then(function() {
-            var fullPath = path.join(this.project.root, 'tests/blanket-options.js');
```

Comment:
I moved all of this into a blueprint file.

Updating by rerunning `ember g ember-cli-blanket` gives you the familiar Overwrite blanket-options.js [Yndeh] prompt